client/consensus/manual-seal/src/finalize_block.rs

```diff
@@ -1,18 +1,20 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
was there any purpose that we have license change in the PR? It seems the old code was using the old format, maybe it was for a reason?

Reply:
I find this seems to be the latest way of license. It also updating the year to 2020